README.md

```diff
@@ -1,56 +1,116 @@
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase">
+        <img alt="stars" src="https://img.shields.io/badge/dynamic/json?color=blue&label=stars&query=stargazers_count&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase">
+        <img alt="forks" src="https://img.shields.io/badge/dynamic/json?color=blue&label=forks&query=forks&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase" />
+    </a>
+    <a href="https://en.oceanbase.com/docs/community-observer-en-10000000000829617">
+        <img alt="English doc" src="https://img.shields.io/badge/docs-English-blue" />
+    </a>
+    <a href="https://www.oceanbase.com/docs/oceanbase-database-cn">
+        <img alt="Chinese doc" src="https://img.shields.io/badge/文档-简体中文-blue" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase/actions/workflows/compile.yml">
+        <img alt="building status" src="https://img.shields.io/github/actions/workflow/status/oceanbase/oceanbase/compile.yml?branch=master" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase/commits/master">
+        <img alt="last commit" src="https://img.shields.io/github/last-commit/oceanbase/oceanbase/master" />
+    </a>
+    <a href="https://join.slack.com/t/oceanbase/shared_invite/zt-1e25oz3ol-lJ6YNqPHaKwY_mhhioyEuw">
+        <img alt="Join Slack" src="https://img.shields.io/badge/slack-Join%20Oceanbase-brightgreen?logo=slack" />
+    </a>
+</p>
+
+**OceanBase Database** is a distributed relational database. It is developed entirely by Ant Group. OceanBase Database is built on a common server cluster. Based on the [Paxos](https://lamport.azurewebsites.net/pubs/lamport-paxos.pdf) protocol and its distributed structure, OceanBase Database provides high availability and linear scalability. OceanBase Database is not dependent on specific hardware architectures.
+
+# Key features
+* Transparent Scalability
+* Ultra-fast Performance
+* Real-time Operational Analytics
+* Continuous Availability
+* MySQL Compatible
+* Cost Effeciency
+
+See [key features](https://en.oceanbase.com/product/opensource) to learn more details.
+
+# System architecture
 ![image.png](https://cdn.nlark.com/yuque/0/2022/png/25820454/1667369873624-c1707034-471a-4f79-980f-6d1760dac8eb.png)
 
-## Roadmap
+[Learn More](https://en.oceanbase.com/docs/community-observer-en-10000000000829641)
+# Quick start
+
+## How to deploy
+
+### 🔥 Deploy by all-in-one
+
+You can quickly deploy a standalone OceanBase Database to experience with the following commands.
+
+```shell
+# download and install all-in-one package (internet connection is required)
+bash -c "$(curl -s https://obbusiness-private.oss-cn-shanghai.aliyuncs.com/download-center/opensource/oceanbase-all-in-one/installer.sh)"
+source ~/.oceanbase-all-in-one/bin/env.sh
+
+# quickly deploy OceanBase database
+obd demo
+```
```

Comment:
Please add one attention, this command can only be run in Linux.

Reply:
Yes. I think it should be noted at the beginning of this section. The `obd` way is only viable on Linux.